src/horizontal_ie9.jsx

```diff
@@ -1,4 +1,7 @@
-import _ from 'lodash';
+import extend from 'lodash/extend';
+import range from 'lodash/range';
+import invokeMap from 'lodash/invokeMap';
+import sum from 'lodash/sum';
```

Comment:
How would one test if this is the case? I have `webpack-dashboard` and it claims the whole lodash library is being bundled.